app/components/publications/urgency-level-checkbox.js

```diff
@@ -0,0 +1,42 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import CONFIG from 'frontend-kaleidos/utils/config';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
+
+
+export default class UrgencyLevelCheckboxComponent extends Component {
+  @tracked urgencyLevels = null;
+  @tracked checkboxValue;
+
+  @service store;
+
+
+  constructor() {
+    super(...arguments);
+    this.loadData.perform();
+  }
+
+  @task
+  *loadData() {
+    this.urgencyLevels = yield this.store.query('urgency-level', {});
+    this.initializeUrgency(this.args.urgencyLevel);
```

Comment:
die initializeUrgency kan je korter schrijven (vervangen) door dit bij in de loadData method te zetten
```
const urgencyLevel = yield this.args.urgencyLevel;
this.checkboxValue = urgencyLevel.get('uri') === CONFIG.URGENCY_LEVELS.spoedprocedure;
```